readme.md

```diff
@@ -20,26 +20,19 @@ After setting up the repository, head to [[getting-started]] to get familiar wit
 
 To learn more about how to use **Foam**, read the [Recipes](https://foambubble.github.io/foam/recipes/recipes) bubbles of the Foam documentation workspace.
 
-And remember that you can always join our  [Foam community on Discord](https://foambubble.github.io/join-discord/g)!
+And remember that you can always join our [Foam community on Discord](https://foambubble.github.io/join-discord/g)!
 
 ## Using Foam
 
-We've created a few Bubbles (markdown documents) to get you started.
+We've created a few Bubbles (Markdown documents) to get you started.
 
 - [[inbox]] - a place to write down quick notes to be categorized later
 - [[getting-started]] - learn how to use your Foam workspace
 - [[todo]] - a place to keep track of things to do
 
 In the `docs` directory you can find everything you need to learn the basics of Foam.
 
-
-[//begin]: # "Autogenerated link references for markdown compatibility"
-[getting-started]: getting-started.md "Getting Started"
-[inbox]: inbox.md "Inbox"
-[todo]: todo.md "Todo"
-[//end]: # "Autogenerated link references"
-
```

Comment:
this section is actually needed for the wikilinks in this file

Reply:
Thanks for the catch. I feel I must've inadvertently deleted this block.

I took a look at this and interestingly I am not able to restore the block. As soon as a I paste the block back in, it gets deleted. There is some VS Code extension that seems to be doing something here. I will investigate that and get this fixed.

Reply:
I have restored this section. I've also reviewed the entire PR to double check that no other inadvertent removals took place. 

As an aside, the above block was remove automatically by the `foam-vscode` plugin. I had to disable it in order to be able to restore this block. I will raise an issue on the `foam` repo and take a look at this.

Reply:
yes this is a sort of hack, because the foam-template is configured to not have the definitions being generated.

Yet we keep them in the `readme.md` to make it navigable from github.com

So, yeah, it's unfortunate, but good enough for now :)